### ISSUE TITLE
STOP button.

### ISSUE THREAD

**Issue by fernando-inf** [essej/sooperlooper#28]
Hi, why is there no STOP button? There is only one from PAUSE. I think a STOP is necessary to be able to assign a MIDI CC to it. Thanks.

**Comment by essej** [comment]
I think what SL needs is a pause_or_trigger command, where it will pause if it's playing, and trigger if it's not.

**Comment by fernando-inf** [comment]
>I think what SL needs is a pause_or_trigger command, where it will pause if it's playing, and trigger if it's not.

Yes, it would be a good advance to minimize the number of pedals. 
Now that I think about it, what you suggest would be the same as a STOP / PLAY button, good idea.

**Comment by essej** [comment]
Oh, I forgot about the existing solution!  The command called mute_trigger... which mutes if playing, triggers from start if muted.  There you go!

**Comment by fernando-inf** [comment]
>The command called mute_trigger... which mutes if playing, triggers from start if muted. There you go!

I will try.

But doing just one MUTE keep the risk of getting XRUN any longer. The continuous playback even when is in MUTE puts unnecessary strain on the processor.

**Comment by fernando-inf** [comment]
Okay, I'll try mute_trigger.

My project uses Guitarix + Hydrogen + Sooperlooper. Sometimes when the 3 programs are active I have XRUN that feel strong, I suppose then that Hydrogen is the culprit. I use a board with an Intel 1037u (2013), but the idea in the future is to bring the headless project to Rapsberry Pi:

https://github.com/fernando-inf/ARDUINO-GUITARIX-MIDI_PEDALBOARD

Greetings, thanks for responding.
